Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec

```diff
@@ -15,6 +15,7 @@
     <dependencies>
```

Comment:
Same questions here.  This list does not match Xamarin.Facebook.AudienceNetwork/packages.config.